docs/life_pt1.md

```diff
@@ -0,0 +1,293 @@
+
+# 13.1 Well...
```

Comment:
Been thinking about this one - perhaps, instead of it being a regular chapter, it could be labelled "extra"? So, something like this:

![image](https://github.com/user-attachments/assets/d04cae7b-4f9c-4dcc-a3ba-0647c1c1156f)

?

Reply:
I like the idea!

Reply:
Done